tests/Yaapii.Atoms.Tests/IO/TempDirectoryTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.IO;
+using Xunit;
+
+namespace Yaapii.Atoms.IO.Tests
+{
+    public sealed class TempDirectoryTest
+    {
+        [Fact]
+        public void CreatesDirectory()
+        {
+            var path = Path.Combine(Path.GetTempPath(), "SunnyDirectory");
+            try
```

Comment:
why not use your class, with using? 
That saves the try catch.

Reply:
@Meerownymous 
because I don't trust my class in the test.

Reply:
You should.. We have a red test if it doesnt work.. So just use it